emukungu/IOT
# Generate pdfs for the visualizations

#### What does this PR do?
-Generate pdfs for the data visualizations

#### Description of Task to be completed?
-Generate pdfs for the data visualizations

#### How should this be manually tested?
1. Check out to the branch `ft-generate-pdfs`
2. Create a virtual environment for the project, run `virtual env`
3. Install dependencies using `pip install -r requirements.txt`
4. On terminal, run command `python manage.py runserver`

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Postman Documentation
- N/A

#### Screenshots (If applicable)
N/A
